src/pylhe/awkward.py

```diff
@@ -10,12 +10,28 @@ def __dir__():
 
 
 def register_awkward():
-    """Register Awkward behaviors for pylhe."""
+    """This function does nothing and is merely here for backwards compatibility
+    of the pylhe API
 
-    vector.register_awkward()
-    ak.mixin_class(ak.behavior)(Particle)
-    ak.mixin_class(ak.behavior)(Event)
-    ak.mixin_class(ak.behavior)(EventInfo)
+    The registration is done automatically when pylhe is imported since awkward
+    is a required dependency of pylhe.
+
+    .. deprecated:: 0.5.3
+       Simply remove calls since the registration happens automatically now.
+    .. warning:: :func:`~pylhe.register_awkward` will be removed in
+     ``pylhe`` ``v0.7.0``.
```

Comment:
```suggestion
    .. deprecated:: 0.6.0
       Remove use of :func:`~pylhe.register_awkward` as registration is automatic.
    .. warning:: :func:`~pylhe.register_awkward` will be removed in
     ``pylhe`` ``v0.8.0``.
```

This is a change in behavior and so not something that should go into a patch release (also `pylhe` is currently `v0.5.1` not `v0.5.2`).